storage/tianmu/handler/ha_my_tianmu.cpp

```diff
@@ -154,6 +154,14 @@ bool ha_my_tianmu_load(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void
   }
   return false;
 }
+bool ha_my_tianmu_is_tianmu_table(std::string db_name, std::string table_name) {
```

Comment:
we should use table's handler to get which engine it belongs to.  NOT just only by extension name or something else.  AND if the active directory is NOT set to tianmu data or set to somewhere else. the stat will failed.  Overall, it NOT a good choice from my point of view. 
```
  // Notify the storage engines that the tables should be made ready for export.
  for (TABLE_LIST *table_list= all_tables; table_list;
       table_list= table_list->next_global)
  {
    handler *handler_file= table_list->table->file;
    int error= handler_file->extra(HA_EXTRA_EXPORT);
    if (error)
    {
      handler_file->print_error(error, MYF(0));
      return true;
    }
  }
```

Reply:
and how about this case. : (1) using a network file system or OBS, etc... 

stat info: 
https://man7.org/linux/man-pages/man2/lstat.2.html

Reply:
OK, thanks very much. I will check this and optimize these code.